spec/controllers/v0/sessions_controller_spec.rb

```diff
@@ -316,6 +316,7 @@ def verify_session_cookie
             levenshtein_distance: 8
           }
         )
+        expect(Raven).to receive(:tags_context).twice
```

Comment:
why twice?

Reply:
`Raven.tags_context` is called in `ApplicationController#set_tags_and_extra_context`
and `SessionsController#saml_callback`